tests/phpunit/CRM/Financial/BAO/PaymentProcessorTypeTest.php

```diff
@@ -116,6 +119,7 @@ public function testDel() {
       'title' => 'Test Del Payment Processor',
       'billing_mode' => 1,
       'is_active' => 1,
+      'class_name' => 'CRM_Core_Payment_Dummy',
```

Comment:
In the DB class_name is stored as `Payment_Dummy` - shouldn't it be here too?

Reply:
Yes I think so